InvenTree/InvenTree/fields.py

```diff
@@ -23,26 +22,28 @@ class InvenTreeRestURLField(RestURLField):
     """Custom field for DRF with custom scheme vaildators."""
     def __init__(self, **kwargs):
         """Update schemes."""
+
+        # Enforce 'max length' parameter in form validation
+        if 'max_length' not in kwargs:
+            kwargs['max_length'] = 200
+
         super().__init__(**kwargs)
         self.validators[-1].schemes = allowable_url_schemes()
 
 
-class InvenTreeURLFormField(FormURLField):
-    """Custom URL form field with custom scheme validators."""
+class InvenTreeURLField(models.URLField):
+    """Custom URL field which has custom scheme validators."""
 
     default_validators = [validators.URLValidator(schemes=allowable_url_schemes())]
 
+    def __init__(self, **kwargs):
+        """Initialization method for InvenTreeURLField"""
 
-class InvenTreeURLField(models.URLField):
-    """Custom URL field which has custom scheme validators."""
-
-    validators = [validators.URLValidator(schemes=allowable_url_schemes())]
```

Comment:
@SchrodingersGat this probalby breaks custom schemas - have you checked for that?